LFS171x/fabric-material/basic-network/docker-compose.yml

```diff
@@ -0,0 +1,123 @@
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This looks like a duplicate of the fabric-samples/basic-network/docker-compose.yml file. The copyright was removed from this. Should it have been?